docs/content/preview/yugabyte-platform/alerts-monitoring/performance-advisor.md

```diff
@@ -0,0 +1,161 @@
+
+**Fix the problem**
+
+If you see query load skew, contact {{% support-general %}}.
+
+### CPU skew and CPU usage
+
+Advisor monitors CPU use to determine whether any nodes become hot spots. Advisor flags nodes where CPU use on a node is 50% greater than the other nodes in the cluster in the past hour (skew), or CPU use for a node exceeds 80% for 10 minutes (usage). CPU skew recommendation is not raised until node has at least 50% average CPU usage in the last 10 minutes.
+
+CPU skew and high usage can be caused by a number of issues:
+
+- A node has a higher number of connections.
+- A node is receiving a higher number of queries.
+- Set of rows or specific rows are accessed frequently by queries executing across different nodes in the cluster.
+
+Any of these conditions can result in a hot spot.
+
+**Fix the problem**
+
+Review the sharding strategies for your primary and secondary indexes. Consistent hash sharding is better for scalability and preventing hot spots, while range sharding is better for range-based queries.
+
+### Rejected connections
+
+Advisor monitors rejected YSQL connections. Advisor flags nodes, which rejected 1 or more connections in the last hour. By default each YB-TServer can handle up to 300 simultaneous connections. This number can be configured with [ysql-max-connections](../../../reference/configuration/yb-tserver/#ysql-max-connections) YB-Tserver flag.
+
+Rejected connections can be caused by:
+
+- Connection skew - all clients are connected to one YB-TServer.
+- Too many clients or too large connection pool sizes.
+- Connection pooling not used on the client side and too many simultaneous requests.
+- Load balancer configured incorrectly.
+
+**Fix the problem**
+
+Review your YSQL clients configuration to see, if connection pooling is configured and pool sizes are correct. If you need more connections, review YB-TServer memory usage. If enough free memory is present, increase the number of `ysql-max-connections`. Otherwise increase the number of YB-TServers to spread connections across them.
+
+### Hot shard
+
+Advisor monitors per-table read or write statistics on each node. Advisor flags nodes, which processed 8x more reads or writes for a particular table in the last 10 minutes. Hot shard recommendation is not raised until a node processes at least 600 reads or writes during the last 10 minutes.
+
+Hot shard can be caused by:
+
+- Using range sharding with sub-optimal shard configuration.
+- An application reading or writing one particular row too often.
+
+**Fix the problem**
+If range sharding is used, analyze reads or writes and recreate the table with more fair shards distribution across nodes.
```

Comment:
How?

Reply:
But also, mechanically, how do you change the shard distribution - are there commands we can point to, or anything like that